File: ConsentCollector/ConsentCollector.Entities/Consent/Comment.cs
using System;

namespace ConsentCollector.Entities.Consent
{
    public sealed class Comment:Entity
    {
        public Comment(Guid idUser, Guid idSurvey, string text) : base()
        {
            Text = text;
            IdUser = idUser;
            IdSurvey = idSurvey;
        }

        public string Text { get; set; }

        public Guid IdUser { get; set; }

        public Guid IdSurvey { get; set; }

        public User User { get; set; }

        public Survey Survey { get; set; }
    }
}

File: Web/src/app/models/comment.ts
export class Comments{
  id?:string;
  text:string;
  idUser:string;
  idSurvey:string;
  public constructor(text:string,idUser:string,idSurvey:string){
    this.text=text;
    this.idUser=idUser;
    this.idSurvey=idSurvey;
  }
}
